test/verify/check-machines

```diff
@@ -129,6 +129,10 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
     def goToMainPage(self):
         self.browser.click(".machines-listing-breadcrumb li:first-of-type a")
 
+    def backToVmListFromVmPage(self):
```

Comment:
I already have a global function for this: 
`self.goToMainPage`

Reply:
Thanks, will remove this method